src/main/kotlin/org/jetbrains/research/commentupdater/models/config/EmbeddingConfig.kt

```diff
@@ -0,0 +1,17 @@
+package org.jetbrains.research.commentupdater.models.config
+
+import com.beust.klaxon.Json
+
+
+data class EmbeddingConfig(
+    @Json(name="max_code_len")
+    val maxCodeLen: Int,
+    @Json(name="max_nl_len")
+    val maxNlLen: Int,
+    val unk: String,
+    val pad: String,
```

Comment:
please, pick more appropriate names for variables. what is pad? what is unck? what it Nl?